sopel/formatting.py

```diff
@@ -9,6 +9,30 @@
 
 import string
 import sys
+
+
+__all__ = [
```

Comment:
This is missing something important: the `colors` class. That's how plugins fetch color codes, usually.